src/main/java/com/vonage/quickstart/application/UpdateApplication.java

```diff
@@ -31,6 +31,7 @@
 import static com.vonage.quickstart.Util.envVar;
 
 public class UpdateApplication {
+    private static final String NAME = "An Updated Name for an Application";
```

Comment:
Same as other App snippet. Why are we hard-coding this?

Reply:
fixed